ember_debug/render-debug.js

```diff
@@ -29,7 +29,7 @@ export default EmberObject.extend(PortMixin, {
     };
 
     this.profileManager.offProfilesAdded(this, this.sendAdded);
-    this.profileManager.offProfilesAdded(this, this._updateRenderPerformanceAndComponentTree);
```

Comment:
This method doesn't exist on `ember-debug/render-debug` at all. It was removed in https://github.com/emberjs/ember-inspector/commit/47138779d4f6c559bda90a0e53df5838ac6da446#diff-04f0316e4c3ec9f79f8bf1ec48f6aafbL45, but we missed updating the cleanup code with the renamed method.